### PR TITLE
Update README example

### PR DESCRIPTION
**Pull request by janczer**
According to #105

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,27 +48,32 @@ Add the following to your `Cargo.toml`
 ```toml
 [dependencies]
 github-rs = "0.6"
+serde_json = "1.0.3"
```
Comment (mgattozzi) [inline README.md]: Can you change this to just "1.0" so that way we're not out of sync with serde_json at all?

**Comment by mgattozzi** [comment]
That's my only change I need you to make! This is great thanks for taking the time to improve the documentation! It's a big help since it's hard to keep updated.

File: README.md (hunk 1 above)
Comment (janczer) [reply README.md]: Done

**Comment by mgattozzi** [comment]
These are just documentation changes so I'm not going to wait for Travis. Thanks again @janczer. You also mentioned you wanted to help contribute elsewhere. If you look under any of the issues for endpoints there's a lot of work available so pick what you want and I can help you from there if you have questions :D

**Comment by janczer** [comment]
@mgattozzi Thank you :)

**Pull request merged**